sunpy/net/helio/tests/test_helio.py

```diff
@@ -260,11 +260,14 @@ def test_link_test_on_urlerror(mock_link_test):
 
 
 @pytest.mark.remote_data
-@pytest.fixture(scope="session")
+@pytest.fixture
 def client():
-    working_links = list(filter(link_test, webservice_parser()))
-    taverna_link = taverna_parser(working_links[0])[0]
-    return HECClient(taverna_link)
```

Comment:
Looking at it, the client itself should handle this already. If it does not, its a bug in the client.